src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -78,6 +78,28 @@ def get_dsn_content(self, dataset_name):
         response_json = self.request_handler.perform_request("GET", custom_args)
         return response_json
 
+    def create_uss_file(self, file_path, type, mode = 'rwxr-xr-x'):
```

Comment:
The signature of the method should be:
`def create_uss(self, file_path, type, mode=None):`

* `create_uss_file` -> `create_uss` since both files and directories can be created with this method.
* `mode` should default to None and only be defined on the payload if a value is supplied.

Reply:
Okay thanks for the review 😊